src/profile-logic/process-profile.js

```diff
@@ -801,6 +803,56 @@ function _processCounters(
   );
 }
 
+/**
+ * Converts the Gecko profiler overhead into the processed format.
+ */
+function _processProfilerOverhead(
+  geckoProfile: GeckoProfile | GeckoSubprocessProfile,
+  // The overhead data is listed independently from the threads, so we need an index that
+  // references back into a stable list of threads. The threads list in the processing
+  // step is built dynamically, so the "stableThreadList" variable is a hint that this
+  // should be a stable and sorted list of threads.
+  stableThreadList: Thread[],
+  // The timing across processes must be normalized, this is the timing delta between
+  // various processes.
+  delta: Milliseconds
+): ProfilerOverhead | null {
+  const geckoProfilerOverhead: ?GeckoProfilerOverhead =
+    geckoProfile.profilerOverhead_UNSTABLE;
+  const mainThread = geckoProfile.threads.find(
+    thread => thread.name === 'GeckoMain'
```

Comment:
There are multiple types of GeckoMain threads, are we sure this is the correct one?

Reply:
Are there? I thought there is only one `GeckoMain` thread per process. Even if there are, I think getting the first one is  not wrong though.